src/hasp/hasp_dispatch.cpp

```diff
@@ -800,7 +800,11 @@ void dispatch_run_script(const char*, const char* payload, uint8_t source)
     path[0] = '.';
     path[1] = '\0';
     strcat(path, filename);
+#if defined(WINDOWS)
```

Comment:
I think this breaks the path on Arduino

Reply:
Arduino doesn't use this part, there's an `#if ARDUINO` above and this part is outside of it. This applies to PC only since it reads files from the disk using `fstream`, which is not used on Arduino.